src/set_theory/game/pgame.lean

```diff
@@ -902,59 +931,7 @@ by rw [equiv, equiv, neg_le_neg_iff, neg_le_neg_iff, and.comm]
 @[simp] theorem neg_fuzzy_neg_iff {x y : pgame} : -x ∥ -y ↔ x ∥ y :=
```

Comment:
If we remove `_iff` from the other places, then we should also remove them here.